fyle_accounting_mappings/views.py

```diff
@@ -53,7 +54,14 @@ def get_queryset(self):
 
         assert_valid(source_type is not None, 'query param source type not found')
 
-        return Mapping.objects.filter(source_type=source_type, workspace_id=self.kwargs['workspace_id'])
+        if self.request.query_params.get('is3D') == 'true':
```

Comment:
I assume this doesn't break if we don't send `is3D` in the query param

Reply:
Yes, it doesn't. I have tested this too